conda/base/context.py

```diff
@@ -456,6 +461,7 @@ def list_parameters(self):
             'root_dir',
             'skip_safety_checks',
             'subdir',
+            'subdirs',
```

Comment:
These fields remain "hidden" and undocumented for now, until we're ready to expose them to users.